lib/platform/gitlab/index.ts

```diff
@@ -232,6 +232,7 @@ export async function initRepo({
   const repoConfig: RepoResult = {
     defaultBranch: config.defaultBranch,
     isFork: !!res.body.forked_from_project,
+    owner: res.body.namespace?.path,
```

Comment:
Are you sure `namespace` is optional? If yes please mark the property at type as optional too.

Reply:
The API docs aren't clear, and certainly don't tell me which properties we can expect across which versions of GitLab.
It feels safer to me to be able to cope if it's not there